### PR TITLE
Remove reduntant license entries and READMEs

### PR DESCRIPTION
**Pull request by XVilka**
# SQUASH ME

**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

Removed some records in `.reuse/dep5` that correspond to already removed files. Also removed now redundant, outdated and incomplete READMEs in some parts of the code.

It's sad that `reuse lint` didn't caught these removed files so I opened a new issue in their repository: https://github.com/fsfe/reuse-tool/issues/566

**Test plan**

CI is green

### CODE REVIEW

**Review by ret2libc — APPROVED** [review]

File: test/Makefile [hunk 1]
```diff
@@ -1,6 +0,0 @@
-all: bins
```
Comment (wargio) [inline test/Makefile:1]: i know that some people does use this makefile to fetch the testbins.

Comment (ret2libc) [reply test/Makefile:1]: yeah true! me too

Comment (XVilka) [reply test/Makefile:1]: Well, they shouldn't.

**Comment by XVilka** [comment]
@wargio @ret2libc if you really miss that `make` - I recommend creating a shell script instead. Since we removed Makefile-based building, I thought it is only one way to stay consistent - remove it everywhere.

**Pull request merged**